std_msgs/Byte blockId
std_msgs/String result